docs/langchain-retrieval-augmentation.ipynb

```diff
@@ -1,875 +1,1269 @@
-            "                 'surrendered. Italy gained the Trentino-South Tyrol, which '\n",
-            "                 'once was owned by Austria.\\n'\n",
-            "                 '\\n'\n",
-            "                 'Fascist Italy \\n'\n",
-            "                 'In 1922, a new Italian government started. It was ruled by '\n",
-            "                 'Benito Mussolini, the leader of Fascism in Italy. He became '\n",
-            "                 'head of government and dictator, calling himself \"Il Duce\" '\n",
-            "                 '(which means \"leader\" in Italian). He became friends with '\n",
-            "                 'German dictator Adolf Hitler. Germany, Japan, and Italy '\n",
-            "                 'became the Axis Powers. In 1940, they entered World War II '\n",
-            "                 'together against France, Great Britain, and later the Soviet '\n",
-            "                 'Union. During the war, Italy controlled most of the '\n",
-            "                 'Mediterranean Sea.',\n",
-            " 'type': 'Document'}\n",
-            "\n"
-          ]
-        }
-      ],
-      "source": [
-        "from pprint import pprint\n",
-        "\n",
-        "documents = pinecone_vectorstore.similarity_search(\n",
-        "    query=\"Who was Benito Mussolini?\",  # our search query\n",
-        "    k=3  # return 3 most relevant docs\n",
-        ")\n",
-        "\n",
-        "for doc in documents:\n",
-        "    pprint(doc.__dict__)\n",
-        "    print()"
-      ]
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "SMFUgtvbFeLQ"
+   },
+   "source": [
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/pinecone-io/examples/blob/master/docs/gen-qa-openai.ipynb) [![Open nbviewer](https://raw.githubusercontent.com/pinecone-io/examples/master/assets/nbviewer-shield.svg)](https://nbviewer.org/github/pinecone-io/examples/blob/master/docs/gen-qa-openai.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "v0to-QXCQjsm"
+   },
+   "source": [
+    "# Retrieval Augemented Generation with Pinecone, Langchain and OpenAI\n",
+    "\n",
+    "## Fixing LLMs that Hallucinate\n",
+    "\n",
+    "In this notebook, you'll learn one of the most common applications of retrieval augmented generation: giving large language models access to up-to-date information. \n",
+    "\n",
+    "\n",
+    "### Demo Data: Pinecone Documentation\n",
+    "\n",
+    "A great example to use RAG is when augmented LLMs with information that may not exist in their training data. This could private data, internal company information, or data that has been updated post a training cutoff. In our case, many modern LLMs are trained on Pinecone libraries that have since been updated, such as release notes or quickstart guides.\n",
```

Comment:
This reads weird. Should the word "augmented" be "augmenting" or did you mean to say something else?

Reply:
I think you mean to say something like "trained on Pinecone data that has since been updated, such as release notes, quickstart guides, blog posts, docs, and code".  I think it's important to have both code and docs there as that's where the bulk of the errors come from (and blog posts!).

> trained on Pinecone libraries that have since been updated, such as release notes or quickstart guides